### PR TITLE
Fix fedmsg-hub --with-consumers option

### PR DESCRIPTION
**Pull request by larchunix**
Use explicit_hub_consumers list to build consumers list forwarded to
moksha.hub. Up until now, consumers list was always None.

### CODE REVIEW

**Review by ralphbean — APPROVED** [review]
Nice!

**Review by Zlopez — APPROVED** [review]

**Comment by larchunix** [comment]
@ralphbean, @Zlopez: ping, is there anything I must do for this PR to be merged?

**Comment by Zlopez** [comment]
@abompard I think you are now the maintainer of fedmsg. So it should be your call for merging.

**Pull request merged**
